torch/nn/functional.py

```diff
@@ -4587,7 +4593,7 @@ def normalize(input: Tensor, p: float = 2.0, dim: int = 1, eps: float = 1e-12, o
     Args:
         input: input tensor of any shape
         p (float): the exponent value in the norm formulation. Default: 2
-        dim (int): the dimension to reduce. Default: 1
+        dim (Union[int, Sequence[int]]): the dimensions to reduce over. Default: 1
```

Comment:
nit: rather than specifying the formal type here, I think this should follow the wording used by the [`torch.norm()` docs](https://github.com/pytorch/pytorch/blob/6623c4838ede2f04f4708a4335dc3668c97523c1/torch/functional.py#L1484):
```
dim (int, tuple of ints, list of ints): the dimensions to reduce over. Default: 1
```